### PR TITLE
Fix Fantem MultiSensor ZB003-X

### PR DESCRIPTION
**Pull request by dsantos21**
Fix Fantem MultiSensor ZB003-X occupancy and illumination sensor, add tamper detection and new fingerprint for same model.

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -5218,7 +5218,15 @@ const converters = {
         type: 'commandStatusChangeNotification',
         convert: (model, msg, publish, options, meta) => {
             const zoneStatus = msg.data.zonestatus;
-            return {occupancy: (zoneStatus & 1<<2) > 0};
+            return {occupancy: (zoneStatus & 1) > 0};
```
Comment (Koenkk) [inline converters/fromZigbee.js:5221]: `ZB003X_tamper` can be merged with this: `return {occupancy: (zoneStatus & 1) > 0, tamper: (zoneStatus & 4) > 0};`

Comment (dsantos21) [reply converters/fromZigbee.js:5221]: Thanks, complete lack of experience here :). Let me try, and thanks for helping!!!

Comment (dsantos21) [reply converters/fromZigbee.js]: I think I did it. Thanks again! The main function still is occupancy, it that a problem?

**Comment by dsantos21** [comment]
Fix issue #2195.

File: converters/fromZigbee.js (hunk 1 above)
Comment (dsantos21) [reply converters/fromZigbee.js]: Running some checks in my local instance before requesting new review!

Comment (dsantos21) [reply converters/fromZigbee.js]: Done, everything working as expected!

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**